package.json

```diff
@@ -53,7 +53,8 @@
     "build:flow": "echo \"// @flow\n\nexport * from '../src';\" > dist/react-beautiful-dnd.cjs.js.flow",
     "storybook": "start-storybook -p 9002",
     "build-storybook": "build-storybook -c .storybook -o site",
-    "prepublishOnly": "yarn build"
+    "prepublishOnly": "yarn build",
+    "prepare": "yarn build:clean && yarn build:dist && yarn build:flow"
```

Comment:
So, this `prepare` script is automatically run on `yarn install` to create the `dist` folder?

Reply:
Yes that is correct!

Reply:
Dropping a link here for reference: https://docs.npmjs.com/cli/v7/using-npm/scripts#life-cycle-scripts